packages/eth-edge/src/controllers/transactions.controller.ts

```diff
@@ -13,4 +17,10 @@ export class TransactionsController {
         this.transactionsService.handleAddressActivityEvent(body);
         return;
     }
+
+    @Get("users/:id/transactions")
+    async findAll(@Param() params: GetTransactionsRequest): Promise<Transaction[]> {
+        const getTransactionReq = await useValidate(getTransactionsSchema, params);
+        return this.transactionsService.findAll(getTransactionReq.id);
```

Comment:
Ideally the services would take the request object like what is done in the api but i noticed the other `findAll` are done like this so its fine for now

```suggestion
        return this.transactionsService.findAll(getTransactionReq);
```